backend/src/server.ts

```diff
@@ -0,0 +1,34 @@
+import { insertSingleRow, getRows } from './databasefunctions';
+import express = require('express');
+import { QueryResultRow } from 'pg';
+import cors from 'cors';
+import formidableMiddleware = require('express-formidable');
+
+// Create a new express application instance
+const app: express.Application = express();
+app.use(formidableMiddleware());
+app.use('*', cors());
+
```

Comment:
I can't see any form of error handling.

Unless it is handled implicitly by Express I recommend adding a centralized error handler so that correct responses can be sent to the front end in case something goes wrong.

If I am not mistaken, in Koa is it something as simple as `app.on('error'..... //handling received error)`. Since Express is made by the same people I am assuming that there is something similar for express

Reply:
I've added a central error handler now, it's slightly more complicated than for Koa, or i might have implemented it in a harder way.